src/transformers/optimization.py

```diff
@@ -519,7 +519,7 @@ def scheduler_hook(param):
             if param.requires_grad:
                 param.register_post_accumulate_grad_hook(scheduler_hook)
 
-        return LayerWiseDummyScheduler()
+        return LayerWiseDummyScheduler(optimizer_dict=optimizer_dict, lr=optimizer.defaults.get("lr", 1e-3))
```

Comment:
It is like a double fallback. Shouldn't be necessary since the dummy optimizer is guaranteed to have a value.

The 1e-3 itself comes from torch galore as their specific defaults.